driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/internal/asserts/DefaultStringAssertion.java

```diff
@@ -99,26 +101,42 @@ public String createSubject() {
 
     @Override
     public BinaryAssertion<Boolean> hasWords(List<String> words) {
-        final String wordsList = String.join("|", words);
-        final Pattern wordsPattern = Pattern.compile("\\b(" + wordsList + ")\\b", Pattern.CASE_INSENSITIVE | Pattern.MULTILINE);
+
+        AtomicInteger found = new AtomicInteger();
+        words.forEach(word -> {
```

Comment:
Also `Collection.forEach()` needs that.

Reply:
Mmh. That would be `Iterable.forEach` and _that_ is also used in the classic for-each loop. As far as I can see nothing _forbids_ it to be parallel, but it would be quite suprising. Ah well, might as well leave it in.